pkg/util/trace.go

```diff
@@ -68,12 +69,13 @@ func CombineTraceProtos(traceA, traceB *tempopb.Trace) (*tempopb.Trace, int, int
 	spanCountTotal := 0
 
 	h := fnv.New32()
+	buffer := make([]byte, 4)
```

Comment:
Just mentioning for reference (was wondering why we need 4bytes), the span kind field is a int32 
https://github.com/open-telemetry/opentelemetry-collector/blob/51661b8400d8a90e1bded85e2770fd56865761be/internal/data/protogen/trace/v1/trace.pb.go#L34

Reply:
32 / 8 = 4?

Reply:
lol i meant i didn't think we need more than 1 byte